docker/sds-api/spine-directory-service/sds/request/formatter_urls.py

```diff
@@ -0,0 +1,15 @@
+class FormatterUrls:
+    SCHEMA_LOCATION_URL = "http://www.w3.org/2001/XMLSchema-instance"
+    ATTR_QNAME_URL = "http://hl7.org/fhir file:///C:/Validation-Assets-R4/Schemas/fhir-all-xsd/fhir-all.xsd"
```

Comment:
does not look legit: file:///C:/Validation-Assets-R4/Schemas/fhir-all-xsd/fhir-all.xsd